packages/react-spinner/src/stories/SpinnerLabel.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+import { Label } from '@fluentui/react-label';
+import { makeStyles, shorthands } from '@griffel/react';
+import { Spinner } from '../index';
+
+const useStyles = makeStyles({
+  container: {
+    '> div': { ...shorthands.padding('20px') },
+    '& label': { display: 'block', marginBottom: '10px' },
+  },
+});
+
+export const Labels = () => {
+  const styles = useStyles();
+  return (
+    <div className={styles.container}>
+      <Spinner labelPosition="before" label={<Label>Label Position Before...</Label>} />
```

Comment:
You should just put the label text here. No need to the `<Label>` since the `label` slot renders that for you. Doing it the way you currently have results in nested `<Label>`s

<img width="363" alt="Nested labels in dev tools" src="https://user-images.githubusercontent.com/93940821/160949245-e5030c58-bbd5-4530-bfc0-e1cbb57d0030.png">